jetstream/cli.py

```diff
@@ -522,6 +551,47 @@ def rerun_config_changed(
         sys.exit(0 if success else 1)
 
 
+@cli.command("ensure_enrollments")
```

Comment:
New CLI command called as part of the Argo workflow. Ensures that enrollment tables are available if the enrollment period is over.

Reply:
Maybe AnalysisExecutor grows a "ensure_enrollments" method.